pkg/sql/catalog/schematelemetry/schema_telemetry_test.go

```diff
@@ -190,4 +190,11 @@ UPDATE system.namespace SET id = %d WHERE id = %d;
 	entries, err := log.FetchEntriesFromFiles(0, math.MaxInt64, 1000, errorRE, log.SelectEditMode(false, false))
 	require.NoError(t, err)
 	require.Len(t, entries, 9)
+
+	// Verify that the log entries have redaction markers applied by checking one
+	// of the specific error messages.
+	errorRE = regexp.MustCompile(`found invalid object with ID \d+: relation ‹"nojob"›`)
+	entries, err = log.FetchEntriesFromFiles(0, math.MaxInt64, 1000, errorRE, log.SelectEditMode(false, true /* keepRedactable */))
```

Comment:
supernit: could we also annotate the first argument to SelectEditMode with /* redact */

Reply:
done